Makefile

```diff
@@ -170,7 +170,7 @@ test: generate-mocks lint generate-deepcopy bin/ginkgo bin/kubectl bin/kube-apis
 		./hack/testing_ginkgo_recover_statements.sh --remove; exit $$EXIT_STATUS
 	
 .PHONY: generate-mocks
-generate-mocks: bin/mockgen $(shell find ./pkg/mocks -type f -name "mock*.go") ## Generate mocks needed for testing. Primarily mocks of the cloud package.
+generate-mocks: bin/mockgen pkg/mocks/mock_client.go $(shell find ./pkg/mocks -type f -name "mock*.go") ## Generate mocks needed for testing. Primarily mocks of the cloud package.
```

Comment:
pkg/mocks/mock_client.go should be returned from $(shell find ./pkg/mocks -type f -name "mock*.go")
Is this additional dependency really required?

Reply:
It's not found from that if the file doesn't exist to begin with.